src/api/user/index.js

```diff
@@ -53,17 +54,17 @@ router.get('/auth/validate/:userName', async (req, res) => {
       return;
     }
     const { code: userCodeInput } = req.query;
-    console.log(`try validate user ${userName} entered correct pin code`);
+    logger.info(`try validate user ${userName} entered correct pin code`);
     await CacheService.init();
     const correctCode = await CacheService.getKey(userName);
-    console.log(`user code ${userCodeInput} correct code ${correctCode}`);
+    logger.info(`user code ${userCodeInput} correct code ${correctCode}`);
     if (correctCode !== userCodeInput) {
-      console.log('wrong code');
+      logger.info('wrong code');
```

Comment:
logger.error

Reply:
i did warn, not really a bug if user entered wrong code